src/coordinator/test/handler.go

```diff
@@ -21,3 +29,32 @@ func (h *SlowHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	time.Sleep(h.delay)
 	h.handler.ServeHTTP(w, r)
 }
+
+// GeneratePromReadRequest generates a sample prometheus remote read request
```

Comment:
Since we only use this in one test package, may be better to put them there?

Reply:
oh yeah, I changed some tests later back.